Modulation

scott-emery opened the pull request:
TAI - Add modulation format attribute to network interface object

This patch adds the modulation format as an attribute of the network interface object with an initial set of supported modulations.

Signed-off-by: Scott Emery <scotte@cumulusnetworks.com>
Fixes a portion of #6

Commit:
Merge pull request #1 from Telecominfraproject/master

Merge from Telecominfroproject master

Commit:
TAI - Add modulation format attribute to network interface object

This patch adds the modulation format as an attribute of the network interface
object with an initial set of supported modulations.

Signed-off-by: Scott Emery <scotte@cumulusnetworks.com>

ishidawataru commented on inc/tainetworkif.h:
```diff
@@ -67,9 +67,26 @@ typedef enum _tai_network_interface_tx_grid_spacing_t
 } tai_network_interface_tx_grid_spacing_t;
 
+/** @brief The modulation formats */
+typedef enum _tai_network_interface_modulation_format_t
```
please add 32_QAM

https://github.com/Telecominfraproject/OOPT-DTC-TAI/issues/6#issuecomment-386872712

Commit:
TAI - Add 32-qam modulation formats to network interface object

This patch adds the 32-qam modulation formats to the network interface object.

Signed-off-by: Scott Emery <scotte@cumulusnetworks.com>

ishidawataru commented:
thank you!
